samples/Nancy/Nancy.Client/project.json

```diff
@@ -0,0 +1,33 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "OpenID Connect extensions for Owin.Security.OpenIdConnect.Server.",
```

Comment:
Woooops :smile: 
You can remove the description, it's obviously useless.

You can also remove the authors and the owners properties :+1: